<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>music player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/musical-note.png</normaloff>images/musical-note.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>551</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(242, 248, 253);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>11</y>
       <width>521</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(5, 0, 33);
color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/back.png</normaloff>images/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="forward_button">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/forward.png</normaloff>images/forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="play_button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/play.png</normaloff>images/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pause_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/pause.png</normaloff>images/pause.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/stop.png</normaloff>images/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="volume_meter">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/volume.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="timeSlider">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>300</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>399</x>
       <y>220</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>230</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelTimer">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>285</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00 / 00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_Songs">
    <property name="title">
     <string>Add Songs</string>
    </property>
    <addaction name="actionAddOneSong"/>
    <addaction name="actionAddManySongs"/>
   </widget>
   <widget class="QMenu" name="menuRemove_Songs">
    <property name="title">
     <string>Remove Songs</string>
    </property>
    <addaction name="actionDeleteASong"/>
    <addaction name="actionDeleteAllSongs"/>
   </widget>
   <addaction name="menuAdd_Songs"/>
   <addaction name="menuRemove_Songs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddOneSong">
   <property name="text">
    <string>Add One Song To Playlist</string>
   </property>
  </action>
  <action name="actionDeleteASong">
   <property name="text">
    <string>Delete A Song From Playlist</string>
   </property>
  </action>
  <action name="actionDeleteAllSongs">
   <property name="text">
    <string>Delete All Songs From Playlist</string>
   </property>
  </action>
  <action name="actionAddManySongs">
   <property name="text">
    <string>Add Many Songs To Playlist</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
